insights/collect.py

```diff
@@ -214,7 +214,7 @@ def get_pool(parallel, kwargs):
         yield None
 
 
-def collect(manifest=default_manifest, tmp_path=None, compress=False):
+def collect(manifest=default_manifest, tmp_path=None, compress=False, rm_conf={}):
```

Comment:
@gravitypriest I think it'd be better to avoid mutable default args like`rm_conf`, maybe this or something similar would be better:
```
def colllect (..., rm_conf=None):
    ...
   rm_conf = rm_conf or {}
```